phase1/vsphere/README.md

```diff
@@ -20,25 +22,24 @@ The deployment is tested with kubernetes v1.4.0, v1.4.4 and v1.4.7
 
 vSphere Cloud Provider is tested on v1.4.7
 
+### Upload VM image to be used to vSphere:
 
-### Upload Template to vSphere:
-You **must** upload the template OS ova to vCenter before deploying kubernetes.
+Upload the template OS OVA to vCenter before deploying kubernetes. All kubernetes nodes will be clones of this VM.
 
 #### Upload using vSphere Client.
+
 1. Login to vSphere Client.
 2. Right-Click on ESX host on which you want to deploy template.
 3. Select ```Deploy OVF template```.
 4. Copy and paste URL for [OVA](https://storage.googleapis.com/kubernetes-anywhere-for-vsphere-cna-storage/KubernetesAnywhereTemplatePhotonOS.ova).
-5. Check the name of the VM created, this will be used to deploy kubernetes later. (Should default to KubernetesAnywhereTemplatePhotonOS)
-6. Follow next steps according to instructions mentioned in wizard. Select the resource pool in which the kubernetes cluster is to be created.
+5. **Check the name of the VM created**, this will be used to deploy kubernetes later. (Should default to KubernetesAnywhereTemplatePhotonOS.ova)
+6. Follow next steps according to instructions mentioned in wizard. Select the resource pool in which the kubernetes cluster is to be created. **Note the resource pool selected** it will be used when deploying kubernetes during ```make deploy``` or ```make config```
```

Comment:
Select the Cluster or Resource Pool in which the kubernetes cluster is to be created. Follow next steps according to instructions mentioned in wizard.
**Note: Resource Pool or Cluster** selected here will be used to deploy Kubernetes during  ```make deploy``` or ```make config```